### PR TITLE
adding a schema to pfe-icon-panel

### PR DESCRIPTION
**Pull request by kylebuch8**

### CODE REVIEW

File: elements/pfe-icon-panel/src/pfe-icon-panel.json [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        }
+      }
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "icon": {
+          "title": "Icon",
+          "type": "string"
```
Comment (castastrophe) [inline elements/pfe-icon-panel/src/pfe-icon-panel.json]: Can you add `"prefixed": false` to these attributes so it's clear they're unprefixed?

File: elements/pfe-icon-panel/src/pfe-icon-panel.json [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          "enum": [
+            "lightest",
+            "lighter",
+            "base",
+            "darker",
+            "darkest",
+            "complement",
+            "accent"
+          ],
+          "default": "base"
```
Comment (castastrophe) [inline elements/pfe-icon-panel/src/pfe-icon-panel.json]: @starryeyez024 Do you think the default background color for a circled icon should be gray or would it make sense in this case to make it complement or accent?

File: elements/pfe-icon-panel/src/pfe-icon-panel.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        },
+        "centered": {
+          "title": "Centered",
+          "type": "Boolean",
```
Comment (castastrophe) [inline elements/pfe-icon-panel/src/pfe-icon-panel.json]: Types in schema are weirdly lowercase 🤷‍♀

File: elements/pfe-icon-panel/src/pfe-icon-panel.json [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        },
+        "stacked": {
+          "title": "Stacked",
+          "type": "Boolean",
+          "default": false
+        },
+        "circled": {
+          "title": "Circled",
+          "type": "Boolean",
+          "default": false
```
Comment (castastrophe) [inline elements/pfe-icon-panel/src/pfe-icon-panel.json]: @starryeyez024 Do you think this should this be true too for the default?

Comment (starryeyez024) [reply elements/pfe-icon-panel/src/pfe-icon-panel.json]: Hmm, good question! I'm thinking that usually we opt into flair, so maybe the circle should be an add-on.

File: elements/pfe-icon-panel/src/pfe-icon-panel.json (hunk 2 above)
Comment (starryeyez024) [reply elements/pfe-icon-panel/src/pfe-icon-panel.json]: I'll echo the same idea, that bright colors are flair, so let's keep it basic (gray) by default.

File: elements/pfe-icon-panel/src/pfe-icon-panel.json (hunk 4 above)
Comment (castastrophe) [reply elements/pfe-icon-panel/src/pfe-icon-panel.json]: 👍

**Pull request merged**